pkg/controllers/nodepool/hash/controller.go

```diff
@@ -80,10 +80,10 @@ func (c *Controller) Builder(_ context.Context, m manager.Manager) operatorcontr
 	)
 }
 
-// Updating `nodepool-hash-version` annotation inside the controller means a breaking change has made to the hash function calculating
-// `nodepool-hash` on both the NodePool and NodeClaim, automatically making the `nodepool-hash` on the NodeClaim different from
-// NodePool. Since we can not rely on the hash on the NodeClaims, we will need to re-calculate the hash and update the annotation.
-// Look at designs/drift-hash-version.md for more information.
+// Updating `nodepool-hash-version` annotation inside the karpenter controller means a breaking change has made to the hash calculation.
+// `nodepool-hash` annotation on the NodePool will be updated, making the `nodepool-hash` on the NodeClaim different from
+// NodePool. Since we can not rely on the `nodepool-hash` on the NodeClaims, due to the breaking change, we will need to re-calculate the hash and update the annotation.
```

Comment:
```suggestion
// NodePool. Since we cannot rely on the `nodepool-hash` on the NodeClaims, due to the breaking change, we will need to re-calculate the hash and update the annotation.
```